sql/index/pilosa/driver.go

```diff
@@ -183,12 +183,25 @@ func (d *Driver) Save(ctx context.Context, i sql.Index, iter sql.IndexKeyValueIt
 	idx.mapping.open()
 	defer idx.mapping.close()
 
+	log := logrus.WithFields(logrus.Fields{
+		"id":     i.ID(),
+		"driver": DriverID,
+	})
+
+	log.Debug("starting to save index data")
+
+	var savedRows uint64
 	for colID := uint64(0); err == nil; colID++ {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 
 		default:
+			if savedRows%100 == 0 {
+				log.Debugf("still creating index: %d rows saved so far", savedRows)
```

Comment:
Can you add some information about the index id and driver id? something like:

Reply:
It's a few lines before, it's using `log`, which is defined here: https://github.com/src-d/go-mysql-server/pull/232/files#diff-619845d6f18affa4db50cfe631210577R186